<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>631</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabUser">
       <attribute name="title">
        <string>User</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>52</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>New user:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>User list:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>511</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Select your user name or create a new one.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonSelect">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>140</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>331</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEditNewUser">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonAdd">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QListWidget" name="userList">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>251</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Train</string>
       </attribute>
       <widget class="QGroupBox" name="MajorImageBox_tr">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>40</y>
          <width>341</width>
          <height>331</height>
         </rect>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QLabel" name="MajorImageLabel_tr">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>256</width>
           <height>256</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="LastButton">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>400</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;&lt;&lt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="NextButton">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>400</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>&gt;&gt;&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>281</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Please draw the gesture with your wiimote. </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>281</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>NOTE:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>281</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>1. When drawing gestures, press button 1.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>180</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>When drawing over, release it.</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabCreateLib">
       <attribute name="title">
        <string>Template</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>301</width>
          <height>421</height>
         </rect>
        </property>
        <property name="title">
         <string>Create template from users</string>
        </property>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>219</width>
           <height>81</height>
          </rect>
         </property>
         <property name="title">
          <string>Creating template methods</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="radioButtonMethodA">
              <property name="text">
               <string>Method A</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonMethodR">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Method R</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>282</width>
           <height>231</height>
          </rect>
         </property>
         <property name="title">
          <string>User list</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QListWidget" name="libUserList"/>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="buttonStart">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>380</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>10</y>
          <width>291</width>
          <height>401</height>
         </rect>
        </property>
        <property name="title">
         <string>Template list</string>
        </property>
        <widget class="QListWidget" name="TemplateList">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>256</width>
           <height>311</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="LoadButton">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>360</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="Recognition">
       <attribute name="title">
        <string>Recognize</string>
       </attribute>
       <widget class="QGroupBox" name="MajorImageBox">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>30</y>
          <width>341</width>
          <height>331</height>
         </rect>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QLabel" name="MajorImageLabel">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>256</width>
           <height>256</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="MajorImageText">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>290</y>
           <width>221</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
       <widget class="QToolBox" name="GestureBoxRecog">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>141</width>
          <height>451</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Page 1</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="page_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>141</width>
           <height>399</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Page 2</string>
         </attribute>
        </widget>
       </widget>
       <widget class="QPushButton" name="buttonSort">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>492</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Sort</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="GestureSaveButton">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>390</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>470</y>
          <width>521</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>If the gesture above is not what you want,</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>490</y>
          <width>511</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string> please select it from left gesture list.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="GestureSelectButton">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>170</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>&gt;&gt;&gt;</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>390</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Finish</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>About</string>
       </attribute>
       <widget class="QLabel" name="AboutLabel">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>551</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Accelerometer based Gesture Recognition Engine</string>
        </property>
       </widget>
       <widget class="QLabel" name="MajorImageText_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>320</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Proudly present by: Wu, Bin</string>
        </property>
       </widget>
       <widget class="QLabel" name="MajorImageText_3">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>230</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Version: 0.80</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
